Src/BootCamp.Chapter/Shop.cs

```diff
@@ -31,6 +24,7 @@ public Item[] GetItems()
         /// </summary>
         public void Add(Item item)
         {
+            Items.Add(item);
```

Comment:
I am sorry for misleading you in my previous review :(! I did not realise the shop already had an inventory field; i thought this was something you added.

If you could pop it back in that'd be appreciated. Then similarly with the Player, Add/Remove will just call into Inventorties Add/Remove

Reply:
done